.github/workflows/release.package.yml

```diff
@@ -57,9 +57,9 @@ jobs:
 
       - name: Git push
         run: COMMIT_MSG=$(npm run --silent package-version --workspace=${{ inputs.working_directory }}) && git commit -am "${{ inputs.package_name }} $COMMIT_MSG" && git push origin main
-
+01
       - name: Release Package
         env:
           NODE_AUTH_TOKEN: ${{ secrets.npm_token }}
-        run: npm publish
+        run: npm publish --access public
```

Comment:
without this flag the very first publish will fail

Reply:
Good catch! thanks Igor 🙇

Reply:
@CarlosGamero tbf, that's on npm. it could have picked up the public access setting from package.json, but noooooo